wd_helper.au3

```diff
@@ -1889,9 +1889,12 @@ Func _WD_ElementActionEx($sSession, $sElement, $sCommand, $iXOffset = Default, $
 
 	Switch $iActionType
 		Case 1
-			$sAction = StringFormat($sActionTemplate, $sPreAction, $iXOffset, $iYOffset, $sElement, $sElement, $sPostHoverAction, $sPostAction) ; StringFormat() usage is significantly faster than building JSON string each time from scratch
+			; StringFormat() usage is significantly faster than building JSON string each time from scratch
+			; StringReplace() removes all possible @TAB's because they was used only for indentation and are not needed in JSON string
+			$sAction = StringReplace(StringFormat($sActionTemplate, $sPreAction, $iXOffset, $iYOffset, $sElement, $sElement, $sPostHoverAction, $sPostAction), @TAB, '')
```

Comment:
- Would the StringFormat comment be placed with the other comments above where we define sActionTemplate?
- Is this StringReplace necessary? I thought this was already handled when the static variable is declared?

Reply:
> * Is this StringReplace necessary? I thought this was already handled when the static variable is declared?

It strips all the possible Tab's from `__WD_JsonAction`

Reply:
> * Would the StringFormat comment be better placed with the other comments above where we define sActionTemplate?

I would prefer to leave this comment in the place where she is. 
Especailly if we remove `StringReplace` from static, and also when we leave the comment about `StringReplace`

Reply:
> * Would the StringFormat comment be better placed with the other comments above where we define sActionTemplate?

On the other hand why not.
Especially to not pollute simplicity of the >Switch ... EndSwitch<
;)

Reply:
> We can remove StringReplace from Static declaration.

What's wrong with leaving it like it was originally written (in the Static declaration)?